doc/DeveloperGuide.md

```diff
@@ -58,7 +58,9 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
+`* * *` | user | edit a person | edit the phone number and/or email address of an entry
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * *` | user | tag an entry with keywords | categories entries into groups by associating each of them with a tag
```

Comment:
Is this really a 3 star User Story? I guess that depends on what user group your application is targeting.